crates/runtime_crypto/src/lib.rs

```diff
@@ -123,6 +124,19 @@ pub fn extract_algorithm_object(
 
             return Ok(Algorithm::AesCtr(counter, length));
         }
+
+        if name == "RSA-OAEP" {
+            let label_key = v8_string(scope, "label").into();
+            let label = match algorithm.get(scope, label_key) {
+                Some(label) => match label.is_uint8_array() {
+                    false => None,
+                    true => Some(extract_v8_uint8array(label)?),
+                },
```

Comment:
I think we should throw an error if `label` isn't an `Uint8Array`

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/RsaOaepParams
emm..., label is optional.

Reply:
Yes it's optional, but if we provide a `label` that isn't an `Uint8Array` we should throw an error (and do nothing if we don't provide a `label`, but that's already the case line 130)